internal/manager/task_stash_box_tag.go

```diff
@@ -258,6 +179,87 @@ func (t *StashBoxPerformerTagTask) stashBoxPerformerTag(ctx context.Context) {
 	}
 }
 
+func (t *StashBoxPerformerTagTask) getPartial(performer *models.ScrapedPerformer, excluded map[string]bool) models.PerformerPartial {
+	partial := models.NewPerformerPartial()
+
+	if performer.Aliases != nil && !excluded["aliases"] {
+		partial.Aliases = &models.UpdateStrings{
+			Values: stringslice.FromString(*performer.Aliases, ","),
+			Mode:   models.RelationshipUpdateModeSet,
+		}
+	}
+	if performer.Birthdate != nil && *performer.Birthdate != "" && !excluded["birthdate"] {
+		value := getDate(performer.Birthdate)
+		partial.Birthdate = models.NewOptionalDate(*value)
+	}
+	if performer.CareerLength != nil && !excluded["career_length"] {
+		partial.CareerLength = models.NewOptionalString(*performer.CareerLength)
+	}
+	if performer.Country != nil && !excluded["country"] {
+		partial.Country = models.NewOptionalString(*performer.Country)
+	}
+	if performer.Ethnicity != nil && !excluded["ethnicity"] {
+		partial.Ethnicity = models.NewOptionalString(*performer.Ethnicity)
+	}
+	if performer.EyeColor != nil && !excluded["eye_color"] {
+		partial.EyeColor = models.NewOptionalString(*performer.EyeColor)
+	}
+	if performer.FakeTits != nil && !excluded["fake_tits"] {
+		partial.FakeTits = models.NewOptionalString(*performer.FakeTits)
+	}
+	if performer.Gender != nil && !excluded["gender"] {
+		partial.Gender = models.NewOptionalString(*performer.Gender)
+	}
+	if performer.Height != nil && !excluded["height"] {
+		h, err := strconv.Atoi(*performer.Height)
+		if err == nil {
+			partial.Height = models.NewOptionalInt(h)
+		}
+	}
+	if performer.Weight != nil && !excluded["weight"] {
+		w, err := strconv.Atoi(*performer.Weight)
+		if err == nil {
+			partial.Weight = models.NewOptionalInt(w)
+		}
+	}
+	if performer.Instagram != nil && !excluded["instagram"] {
+		partial.Instagram = models.NewOptionalString(*performer.Instagram)
+	}
+	if performer.Measurements != nil && !excluded["measurements"] {
+		partial.Measurements = models.NewOptionalString(*performer.Measurements)
+	}
+	if excluded["name"] && performer.Name != nil {
```

Comment:
Had a question on this in the Studio tagger PR.. why is this one opposite logic vs all the other excluded fields?

Reply:
Looks like the original line was `if excluded["name"]`, so the nil check was added afterwards (in #1630). Makes no functional difference.